schema/schema.go

```diff
@@ -483,6 +483,19 @@ func (schema *Schema) StateVersioning() bool {
 	return stateful
 }
 
+//Disable like search wheter to use substring search or full string search on column values
+func (schema *Schema) SearchLikeDisabled() bool {
```

Comment:
since the property is set in metadata of the schema..we need this helper function to evaluate the 'disable_substr_search' flag and i don't see any other means to access schema definition in sql module other than the schema object loaded and accessing via helpers

Reply:
1. first of all, this check should not be done in the sql package, but rather to the calling code. if a request comes from an external API, it's the middleware responsibility to either allow or reject it with 400 BadRequest. I like the idea of executing LIKE queries, I'd like to do it for example from goext. In goext I've got a full control over the logic, so I do not want the sql package to do anything behind the scenes.
2. prefer a) non-negative (pun intented) logic 2) `IsSomething() bool` naming convention. in this case this method would become `IsFuzzySeachEnabled() bool`